packages/amplify-category-analytics/provider-utils/awscloudformation/cloudformation-templates/pinpoint-cloudformation-template.json

```diff
@@ -183,7 +183,7 @@
                                 "           }",
                                 "       };",
                                 "       const pinpoint = new aws.Pinpoint({ apiVersion: '2016-12-01', region: event.ResourceProperties.region });",
-                                "       return pinpoint.createApp(params).promise()",
```

Comment:
Did you figure out why returning a promise is failing?

Reply:
No, I'm still researching it and might log a bug for either Cloudformation or Lamdba. It was working for the older node version, but for the new node version, you can not return a promise, otherwise the output of the CFN custom resource is not awaited.

Reply:
logged an issue for lambda